index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./output.css">
+    <script src="./script.js" defer></script>
+    <title>Fake OS</title>
+</head>
+<body>
+   <main class="background-image">
```

Comment:
Add an icon to the desktop - it should open a modal with some functionality, for example a weather app, currency conversion, horoscope, etc